resources/prosody-plugins/mod_av_moderation_component.lua

```diff
@@ -0,0 +1,242 @@
+local get_room_by_name_and_subdomain = module:require 'util'.get_room_by_name_and_subdomain;
+local is_healthcheck_room = module:require 'util'.is_healthcheck_room;
+local json = require 'util.json';
+local st = require 'util.stanza';
+local um_is_admin = require "core.usermanager".is_admin;
+
+local muc_component_host = module:get_option_string('muc_component');
+if muc_component_host == nil then
+    log('error', 'No muc_component specified. No muc to operate on!');
+    return;
+end
+
+module:log('info', 'Starting av_moderation for %s', muc_component_host);
+
+local function is_admin(jid)
+    return um_is_admin(jid, module.host);
+end
+
+-- Notifies that av moderation has been enabled or disabled
+-- @param jid the jid to notify, if missing will notify all occupants
+-- @param enable whether it is enabled or disabled
+-- @param room the room
+-- @param actorJid the jid that is performing the enable/disable operation
+-- @param mediaType the media type for the moderation
+function notify_occupants_enable(jid, enable, room, actorJid, mediaType)
+    local body_json = {};
+    body_json.type = 'av_moderation';
+    body_json.enabled = enable;
+    body_json.room = room.jid;
+
+    local notify = function(jid_to_notify, mediaType)
+        body_json.actor = actorJid;
```

Comment:
`actorJid` and `mediaType` are always the same, you could set them once above, and you wouldn't need to pass `mediaType` as a param to `notify`

Reply:
Fixed.